src/runtime/services.js

```diff
@@ -156,7 +156,7 @@ export function feUrl(
       ? `swg/${prefix}`
       : `${prefix}/swg`
     : 'swg';
-  url = feCached(`${getSwgMode().frontEnd}/${prefixed}/_/ui/v1${url}`);
+  url = feCached(`${getSwgMode().frontEnd}/${prefixed}/ui/v1${url}`);
```

Comment:
Wanna just collapse 154-159 since its no longer behind a flag?

Reply:
Is it cool if I tackle this in a followup PR?

Reply:
sgtm